core/arch/arm/include/mm/core_mmu.h

```diff
@@ -239,7 +239,8 @@ struct core_mmu_phys_mem {
 		__register_memory1(#addr, MEM_AREA_SDP_MEM, (addr), (size), \
 				   phys_sdp_mem_section, __COUNTER__)
 #else
-#define register_sdp_mem(addr, size)
+#define register_sdp_mem(addr, size) \
+		static int CONCAT(unused, __COUNTER__) __unused
```

Comment:
We should probably have a prefix to `unused`, how about `__phys_mem_unused` instead?

Reply:
What would it help for? I see no possible name conflict since `__COUNTER__` is used. And the symbol will normally not even make it into the object file. Now, I agree it's always better to be more specific than too vague, with that in mind let's move to my second argument:
`__phys_mem_unused` sounds a bit like it's some unused physical memory as opposed to some unused symbol... How about `register_sdp_mem_unused` maybe?

Reply:
Fine, with a `__` in front.

Reply:
OK, `__` is desirable indeed.